phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/MainWindowView.java

```diff
@@ -74,16 +74,20 @@ protected void setSideBar(SideBar sideBar) {
         this.leftContent.setContent(sideBar);
     }
 
-    public void showRightView(Node nodeToShow) {
+    public void setCenter(Node nodeToShow) {
```

Comment:
I think it would be quite nice if we use this opportunity to add some JavaDoc to this class.
This JavaDoc should describe, among other topics, how the view is structured.